images/win/scripts/Tests-Windows2016/SSDTExtensions.Tests.ps1

```diff
@@ -0,0 +1,6 @@
+Describe "SSDTExtensions" {
```

Comment:
Why do you need to split tests to `Tests-Windows2016` and `Tests-Windows2019`? It brings additional complexity.

You can just use `If (Test-IsWin19)`. See https://github.com/actions/virtual-environments/blob/main/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L376

We already use this approach in tests. See example: https://github.com/actions/virtual-environments/blob/58f3ee85cb41e84a887ec7e9d214fb50e4451155/images/win/scripts/Tests/Android.Tests.ps1#L45